aliases/ansible.yaml

```diff
@@ -3,6 +3,17 @@
 ---
 name: "chatops.ansible_local"
 action_ref: "ansible.command_local"
-description: "Run ansible command on local machine"
+description: "Run Ansible command on local machine"
 formats:
-  - "ansible {{args}}"
+  - display: "ansible <command>"
+    representation:
+      - "ansible {{ args }}"
+result:
+  format: |
+    Ansible command `{{ execution.parameters.args }}` result: {~}
+    {% if execution.result.stderr %}*Stdout:* {% endif %}
+    ```{{ execution.result.stdout }}```
+    {% if execution.result.stderr %}*Stderr:* ```{{ execution.result.stderr }}```{% endif %}
```

Comment:
Not in a way that's easy, but you can take a look on how it's generated in `ack` and replicate it in the template. Getting execution ID for the URL definitely isn't a problem, but I'm not entirely sure the hostname is stored somewhere in metadata.